community/opendkim/APKBUILD

```diff
@@ -3,55 +3,63 @@
 # Maintainer: Valery Kartel <valery.kartel@gmail.com>
 pkgname=opendkim
 pkgver=2.10.3
-pkgrel=4
+pkgrel=5
 pkgdesc="DKIM signing and verifying filter for MTAs"
 url="http://www.opendkim.org"
 arch="all"
 license="custom"
+depends=""
 makedepends="libressl-dev libmilter-dev"
 pkgusers="opendkim"
 pkggroups="opendkim"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dbg"
-install="$pkgname.pre-install"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dbg $pkgname-utils"
+install="$pkgname.pre-install $pkgname.post-install"
 source="http://downloads.sourceforge.net/project/$pkgname/$pkgname-$pkgver.tar.gz
-	opendkim.conf
-	opendkim.initd"
-builddir="$srcdir/$pkgname-$pkgver"
+	$pkgname.confd
+	$pkgname.initd"
+builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
I used old-style apkbuild template. As I see in current one the whole line is quoted.